py/__init__.py

```diff
@@ -54,7 +63,8 @@ def py_get_import_path_holders(param_type):
     "Queue.drainTo",
     "ReplicatedMap.addNearCacheEntryListener",
     "ScheduledExecutor",
-    "Sql",
+    "Sql.execute_reserved",
+    "Sql.fetch_reserved",
```

Comment:
why these needed? I deleted them and I think they are still not generated

Reply:
For your case, they won't be generated because there are missing type mappings for the TypeScript codecs (`List_ListCN_Data`). If you add it, you will see that they will be generated.

You have to add them here to be sure that they won't be generated

Reply:
no i tried python codecs and they are not generated